### DISCUSSION TITLE
Getting ERR_CONNECTION_REFUSED when running service under a dedicated user

### DISCUSSION THREAD

**Discussion by EHRETic** [requarks/wiki#3071] [Help / Questions]
Hi there,

So I discovered Wiki Js and in order to create my own private server, I followed the documentation and was able to get Wiki.js working so far. From now on, I run it on port 80 with user root, but I'll probably put it behind NGINX later on.

I can run the service with User=root, I can run it manually with the command but whenever I try my user "wiki" in the service, I get ERR_CONNECTION_REFUSED even if in the background, the service runs.

I followed this guide (for some parts): https://computingforgeeks.com/how-to-install-wiki-js-on-centos-8/

And more specifically, the following to create/configure the user:
_sudo groupadd --system wiki
sudo useradd -s /sbin/nologin --system -g wiki wiki_

and
_sudo chown -R wiki:wiki /opt/wikijs_

Folder looks like that:
![image](https://user-images.githubusercontent.com/28634810/106309139-fcf20000-6261-11eb-9713-d8dd5729a429.png)

Am I missing something there? Thanks in advance for your help.

Cheers 😉

**Comment by mvaldes14** [comment]
does the error indicate if its having issues connecting to a db or redis or something?
are all services running under their own users?
can you share your service unit details?
